src/components/Select/Select.tsx

```diff
@@ -0,0 +1,92 @@
+"use client"
+
+import {
+  CheckIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+} from "@radix-ui/react-icons"
+import * as RadixSelect from "@radix-ui/react-select"
+import clsx from "clsx"
+import type React from "react"
+import "./styles.css"
```

Comment:
Let's use css modules as it helps to isolate class names.

Reply:
We are considering customizing the theme of our widget so in this case CSS Module wouldn't allow us to use dynamic config. I assume that we have to use CSS-in-JS which is a more robust choice for the kind of customization.